src/error-handler.ts

```diff
@@ -0,0 +1,14 @@
+import { ErrorRequestHandler } from 'express';
+import Logger from './modules/logger/logger';
+
+const serverErrorHandler: ErrorRequestHandler = (err, _req, res, _next) => {
+  const errorCode = err.code as string;
+  const errorMessage = err.message as string;
+  Logger.error(`Error: ${errorCode} ${errorMessage}`);
```

Comment:
We should call `err. toString()`. This way each class can log details they want.